frontend/src/reports/ReportProvider.tsx

```diff
@@ -132,6 +132,9 @@ function ReportProvider(props: { madLib: MadLib; setMadLib: Function }) {
         className={styles.MissingDataInfo}
         ref={fieldRef}
       >
+        <h3> Notes </h3>
```

Comment:
I think we should redo this wording / layout but I can work on it after it's merged in. I think it's important to keep the "What Data Are Missing" larger up top. Then I think we should include the note about 1-dose but phrased as a missing data problem; as in, we would include the "fully" dosed people if they had the breakdown by demographic, correct? The only reason we are using single dosed is because that's the category that includes demographic breakdown. Do we also need to mention that data is missing from the CDC at county level and that we rely on a different data source?

Reply:
maybe the specific details will be on the methodology page I guess? but I still like the idea of keeping this note as a missing data problem if we can, and under the larger heading that's already there

Reply:
should be plural ? 

> "Vaccinated Individuals" refers to people who have received at least one dose of a COVID-19 vaccine.

Reply:
For sure, I do think we need to put a conspicuous note on the page explicitly saying that we are talking about at least one dose though.

Reply:
And yeah I should add a vaccine section to the "what data is missing" , will do that,

Reply:
@benhammondmusic where do you think we should put this note?

Reply:
I'm almost thinking having a large heading (same size as "what data are missing" called "definitions" or something to that effect, and then we define our variables or otherwise clarify what the terminology means. We could also include our note about using inclusive language re "some other race" there later on maybe? 

I think it looks a little strange to have a smaller subheading first for "notes" and then the large heading "what data are missing" below it.

Reply:
I agree with that, but I kind of think it should be somewhere more prominent on the page than in the middle of that large block of text.

Reply:
Yes, definitely. I was thinking up top, like this. We could also consider finding a home higher on the page for important definitions or clarifications if they are essential to understanding the visualizations 

![definitions vaxx](https://user-images.githubusercontent.com/41567007/133158667-ae8f8aac-9ffc-41aa-94f5-2a4691e7d58b.png)